src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -105,22 +109,26 @@ function composeDrillContextPoint({ point }: IHighchartsChartDrilldownEvent, cha
 const chartClickDebounced = debounce(
     (
         drillConfig: IDrillConfig,
-        event: IHighchartsChartDrilldownEvent,
+        event: Highcharts.DrilldownEventObject,
         target: EventTarget,
         chartType: ChartType,
     ) => {
         const { afm, onFiredDrillEvent } = drillConfig;
-
         let usedChartType = chartType;
-        if (isComboChart(chartType)) {
-            usedChartType = get(event, ["point", "series", "options", "type"], chartType);
+        let drillContext: IDrillEventContext;
+
+        if (isGroupHighchartsDrillEvent(event)) {
+            const points: IHighchartsPointObject[] = event.points as IHighchartsPointObject[];
```

Comment:
its necessary to declare type twice ? 
`const points = event.points as IHighchartsPointObject[];` I guess it should be enough